db/README.md

```diff
@@ -109,3 +109,19 @@ Set the preferred maintenance window (For production this is currently every Sun
 ```
 cf update-service htbhf-claimant-service-postgres -c '{"preferred_maintenance_window": "Sun:03:00-Sun:03:30"}'
 ```
+
+Claim auditing
+-----------
+All changes to claims are audited using the [javer](https://javers.org/) library. 
+
+* jv_global_id — domain object identifiers,
+* jv_commit — Javers commits metadata,
+* jv_commit_property — commit properties,
+* jv_snapshot — domain object snapshots
```

Comment:
Don't we need a sql script to create these tables?

Reply:
It's done automatically and seems to be the recommended option. The tables are created if they don't exist. We can change this to create them with flyway if preferred.

Reply:
I think we should be using flyway, rather than relying on tables being created at startup

Reply:
Added flyway script.